src/common/reducers/request-snap-builds.js

```diff
@@ -0,0 +1,34 @@
+import * as ActionTypes from '../actions/request-snap-builds';
+import { snapBuildFromAPI } from '../helpers/snap-builds';
+
+export function requestSnapBuilds(state = {
+  isFetching: false,
+  builds: [],
```

Comment:
There seems to be a bit of redundancy, as we already request and keep builds list in `snapBuilds` reducer (for data fetched for builds list).

As discussed on IRC, let's think about refactoring it later.